cfme/tests/openstack/test_host_power_control.py

```diff
@@ -0,0 +1,46 @@
+import pytest
+from utils import testgen
+from cfme.web_ui import Quadicon
+from cfme.infrastructure.host import Host
+
+pytest_generate_tests = testgen.generate(testgen.provider_by_type,
+                                         ['openstack-infra'], scope='module')
+
+
+@pytest.mark.usefixtures("setup_provider_modscope")
+def test_host_power_off(provider):
```

Comment:
**REQUIRED:** This test should probably have a prerequisite that turns the host on if it is off already. In this way, the test will become a little shorter and will still be tested even if the host is off at the start of it.